### ISSUE TITLE
Class incremental training

### ISSUE THREAD

**Issue by shivSD** [AlexeyAB/darknet#3680]
@AlexeyAB I have trained yolov2 model to detect K objects, now I want to add one more class. So after doing incremental training it should detect K+1 object. I'm using previously trained K weight file as a pretrained weights (training only on new class images without using K old images). Learning rate is in the order of 1e-06. With below CFG file and command line model it is completely forgetting the old K objects.  

**CFG file looks like this:**
learning_rate=0.0000001
max_batches = 65000
filters=80
activation=linear
[region]
anchors = 1.08,1.19,  3.42,4.41,  6.63,11.38,  9.42,5.11,  16.62,10.52
bias_match=1
classes=11

_./darknet detector train  data/obj.data cfg/yolo-obj-Kplsu1.cfg /weights/yolo-obj-K_20000.weights_

Can i train it on K+1 classes just by using weights of K classes, and images of the K+1 th class?
Is there anything missing in CFG file or command line? 

thanks for your advice
